lexers/embedded/rgbasm.xml

```diff
@@ -0,0 +1,238 @@
+<lexer>
+  <config>
+    <name>RGBDS Assembly</name>
+    <alias>rgbasm</alias>
+    <filename>*.asm</filename>
```

Comment:
Mmmm, I think this will conflict with a bunch of other asm syntaxes. Can you add

```xml
<priority>0.5</priority>
```

This will mean it still gets picked up, but takes lower priority.

Reply:
Ah yes, didn't think of that. Done!